scripts/dev_settings.sh

```diff
@@ -16,10 +16,6 @@ if [ -z "$EQ_ENABLE_SECURE_SESSION_COOKIE" ]; then
     export EQ_ENABLE_SECURE_SESSION_COOKIE=False
 fi
 
-if [ -z "$EQ_RABBITMQ_ENABLED" ]; then
-  export EQ_RABBITMQ_ENABLED=False
-fi
-
 if [ -z "$EQ_RABBITMQ_HOST" ]; then
```

Comment:
The convention to date has been to default to "production" settings and then override in dev.
With this setup if we don't set `EQ_SUBMITTER` then nothing will be submitted and we wont get any errors

Reply:
Addressed.